plugins/nsatz/ideal.mli

```diff
@@ -6,6 +6,17 @@
 (*         *       GNU Lesser General Public License Version 2.1        *)
 (************************************************************************)
 
+type metadata = {
+  name_var : string list;
```

Comment:
One could possibly put a debug flag in there. It doesn't cost anything and helps understanding the code thanks to nominal typing.